api-reference/10 UI Components/dxCalendar/1 Configuration/disabledDates.md

```diff
@@ -20,6 +20,12 @@ Information about the checked date.
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
 }
 
+In cases of *'multiple'* and *'range'* [selection modes](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode), the behavior of disabled dates in Calendar is the following:
+
+- If you specify the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property programmatically, disabled dates in the values array are selected.
+
+- If you use UI to change selection (by clicks, Enter key, or [selecting a week on click](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectWeekOnClick)), you cannot select disabled dates in the *'multiple'* mode. In the *'range'* mode, disabled dates cannot be selected as the range start and range end, but can be included inside the range.
```

Comment:
We usually don't use articles with mode names: in Edit mode, in Airplane mode, in Silent mode, etc. 

The rest is a few edits where I try to shorten the phrases a bit. 

```suggestion
- If you use UI to change selection (clicks on dates or [weeks](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectWeekOnClick), the Enter key), you cannot select disabled dates in *'multiple'* mode. In *'range'* mode, disabled dates cannot start or end a range, but can be included in the middle.
```